packages/atlas/src/components/_video/VideoPlayer/ControlsIndicator.tsx

```diff
@@ -51,10 +54,24 @@ export const ControlsIndicator: FC<ControlsIndicatorProps> = ({ player, isLoadin
     const indicatorEvents = Object.values(CustomVideojsEvents)
 
     const handler = (e: Event) => {
+      let selectedLanguage = 'Off'
+      const tracks = player?.remoteTextTracks()
+      for (let i = 0; i < tracks.length; i++) {
+        const track = tracks[i]
+        if (track.mode === 'showing') {
+          selectedLanguage = track.label
+        }
+      }
       // This setTimeout is needed to get current value from `player.volume()`
       // if we omit this we'll get stale results
       timeout = window.setTimeout(() => {
-        const indicator = createIndicator(e.type as VideoEvent, player.volume(), player.muted())
+        const indicator = createIndicator(
+          e.type as VideoEvent,
+          player.volume(),
+          player.muted(),
+          selectedLanguage,
+          player.playbackRate()
+        )
```

Comment:
I'm thinking, maybe we could just pass `player` here? Function with 5 parameters feels awkward